test/fake-intake/Dockerfile

```diff
@@ -0,0 +1,29 @@
+## Based on https://docs.docker.com/language/golang/build-images/
+# syntax=docker/dockerfile:1
+
+## Build
+FROM golang:1.19-buster AS build
+
+WORKDIR /app
+
+COPY go.mod ./
+COPY go.sum ./
+RUN go mod download
+
+COPY fakeintake/*.go ./fakeintake/
+COPY *.go ./
+
+RUN go build -o /build
+
+## Deploy
+FROM gcr.io/distroless/base-debian10
+
+WORKDIR /
+
+COPY --from=build /build /fakeintake
+
+EXPOSE 5000
```

Comment:
`500x` is already used in the Agent, you may to choose something else to avoid confusions.

Reply:
Is there a list of available ports ?

Reply:
Found it ! https://docs.datadoghq.com/agent/basic_agent_usage/?tab=agentv6v7